docs/dcp2_operating_procedures.rst

```diff
@@ -0,0 +1,304 @@
+
+B) (Meta)data exhibiting the proposed schema changes are committed to a feature
+    branch of the `schema-test-data`_ repository, and a PR is filed for that
+    branch. The `schema-test-data`_ repository can only populated using the
+    staging instance of the Ingest component, so the PR against the `develop`
+    branch of the `metadata-schema`_ repository must be approved (under the
+    condition that the schema change may need to be amended in another PR)
+    and the changes are promoted to the staging branch.
+
+The reviewers of a PR can request either testing strategy. Strategy A will
+likely be requested for for most changes with some impact on other components.
+
+In some cases a lab submission not only involves a schema change but also
+introduces a novel way of linking the metadata enties into subgraphs. In those
+cases, reviewers are likely to request testing strategies B. A request for
+either strategy should be reasonably motivated. Possible reasons include 
+
+- the need to not only review schema changes but also review how those changes
+  affect actual metadata documents
+
+- the need to test future, unrelated code changes against the test metadata, so
+  as to make sure that those future changes don't introduce regressions.
+  Especially graph changes fall into that category.
+
+Schema changes that are sized incorrectly need to be revised.
+
+
+Schema change authorship
+------------------------
+
+Anyone can author a schema change. The PRs for one schema change don't all need
+to be by the same author, as long as there are no competing PRs pending in the
+same repository, for the same schema change. One person *should* author the PRs
+against the `metadata-schema`_ and `dcp2`_ repositories. That person is
+referred to as the *schema change author*. The schema change author can
+delegate authorship of PRs to other members of the DCP consortium.
+
+
+Pull request reviews
+--------------------
+
+Pull requests against the `dcp2`_, the `metadata-schema`_ or the
+`schema-test-data` repositories are reviewed and approved in exactly the same
+way, provided they are involved in a semantic change to either the DCP/2
+specification, a DCP/2 standard operating procedure or a metadata schema
+change:
+
+As a PR author
+
+1)  Announce the PR on the ``#dcp2`` channel on Slack, @-mentioning all
+    `Designated reviewers`_.
+
+2)  Request a review from all `Designated reviewers`_.
+
+3)  Wait one week.
+
+4)  If this is the first review cycle, remind about the PR on the ``#dcp2``
+    channel on Slack, @-mentioning requested reviewers that haven't yet
+    reviewed the PR.
+
+5)  If either
+
+    a)  all reviewers approve of the PR without conditions or
+
+    b)  two weeks have passed since step 1 and there are no binding reviews
+        requesting changes,
+
+    merge the PR. You are done unless any of the approvals are conditional. A
+    conditional approval is one that's dependent on successful testing using
```

Comment:
How do we identify conditional approvals? Are we expecting a comment on the PR, a successful review with a comment?

Reply:
Via comment. I'll clarify this.

Reply:
Fixed